src/api/services/timecardService.js

```diff
@@ -5,7 +5,7 @@ const serviceName = 'Timecard Service ';
 
 export const getHelloWorld = async () => {
```

Comment:
Can we remove this hello world ajax call now and the output on the homepage given the new `saveTimeCard` requests proves we can hook up to the back-end?

Reply:
good spot, with pleasure. I have removed it and enjoyed deleting all the code a little bit too much